template-sample-app/template/src/data/models/isochroneModel.js

```diff
@@ -13,7 +13,7 @@ export const RANGES = {
   THIRTY: 30,
 };
 
-export const getIsochrone = async ({
```

Comment:
Same here, I prefer to keep the async declaration, removing the unneeded await

Reply:
I agree